index.js

```diff
@@ -1,15 +1,57 @@
-module.exports = ({
-  user = (ctx) => `You're the following permissions: ${ctx.command.userPerms.join(', ')}`,
-  my = (ctx) => `I'm missing the following permissions: ${ctx.command.myPerms.join(', ')}`
+const humanReadable = {
```

Comment:
Does doing separating the variables work for you? I was not able to get it to work. Starting the cluster would throw the error: `ReferenceError [Error]: Cannot access 'humanReadable' before initialization`

Reply:
The error was thrown on line 36.